src/me/components/LogoutButton.tsx

```diff
@@ -1,20 +0,0 @@
-// Libraries
-import React, {FC} from 'react'
-import {Link} from 'react-router-dom'
-
-// Components
-import {Button, ComponentSize} from '@influxdata/clockface'
-
-const LogoutButton: FC = () => (
-  <>
-    <Link to="/logout">
-      <Button
-        text="Logout"
-        size={ComponentSize.ExtraSmall}
-        testID="logout--button"
-      />
-    </Link>
-  </>
-)
-
-export default LogoutButton
```

Comment:
does this mean we can only logout from that dropdown?

Reply:
Yes, this was a design decision and this is based on the mockup